platform_versioned_docs/version-23.1.0/enterprise/advanced-topics/use-iam-role.mdx

```diff
@@ -13,22 +13,131 @@ AWS-based customers can configure Tower to interact with other AWS Services like
 
 ## Configure the Tower IAM policy
 
-!!! tip "Assumptions in Provided Policies"
-
-    The provided policies were designed with certain assumptions:
+:::note
+The provided policies were designed with certain assumptions:
 
-    1. **IAM Policy**: Tower and Nextflow should have whole access to identified S3 Buckets.
-    2. **Trust Policy**: The Role should be assumable by EC2, ECS, EKS, and only specifically-named IAM actors.
+1. **IAM Policy**: Tower and Nextflow should have whole access to identified S3 Buckets.
+2. **Trust Policy**: The Role should be assumable by EC2, ECS, EKS, and only specifically-named IAM actors.
 
-    You may wish to limit S3 access to specific prefixes, and/or Role assumption to more specific Platforms.
+You may wish to limit S3 access to specific prefixes, and/or Role assumption to more specific Platforms.
+:::
```

Comment:
Ok, this sort of covers my concern above.

Do we need to include new verbiage here about how these permissions impact Data Explorer and (in future) Data Studio(?) functionality?

Reply:
Added an admonition for DE. Will revise it to include DS when it is released.